Mods/Alpakit/Source/Alpakit.Automation/PackagePlugin.cs

```diff
@@ -50,7 +49,7 @@ public override void ExecuteBuild()
 		{
 			var copyToGameDirectory = ParseOptionalDirectoryReferenceParam($"CopyToGameDirectory_{SC.FinalCookPlatform}");
 			var launchGameType = ParseOptionalEnumParam<LaunchGame.LaunchType>($"LaunchGame_{SC.FinalCookPlatform}");
-			var customLaunch = ParseOptionalStringParam($"CustomLaunch_{SC.FinalCookPlatform}");
+			var customLaunch = ParseOptionalStringParam($"CustomLaunchPath_{SC.FinalCookPlatform}");
```

Comment:
Since the argument takes more than just the path, I think the previous name was better (which was probably my intent before)

There's a couple more usages of this name in other files